src/main/java/Ixo.java

```diff
@@ -0,0 +1,151 @@
+import java.util.Scanner;
+
+
+public class Ixo {
+
+    public final static String SEPARATOR = "____________________________________________________";
+
+    public static void menu() {
+        System.out.println("1. Task List");
+    }
+
+    public static void taskList() {
```

Comment:
Similarly for this, function names should be verbs. Also, the function name does not appear to match what is happening inside it. Additionally, thif function is >30LOC. Consider breaking it up into multiple smaller functions.